### PR TITLE
Fix #74: Validate blog URLs.

### LINKED ISSUE #74: Validate blog URLs
Currently the feeder do not validate the blog's URLs. Students will add and modify blogs URLs and there must be a way to check if these new URLs are valid and reachable.  In this issue the objective is to check for valid format of URLs only.

### PR DESCRIPTION
**Pull request by Cadesh**
I tested some ways to validate the URLs, the one that worked better was with valid-url [(https://www.npmjs.com/package/valid-url)](url)

These are the changes to the code:
1) Added a new file utils.js with a function to validade URLs. 
2) Added a file utils.test.js with a test for the new function.

### CODE REVIEW

**Comment by kartik-budhiraja** [comment]
@Cadesh can you please look into the eslint errors, the travis pipeline logs can help you there.

File: src/utils.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ */
+function isValidUrl (url)  {
+    if (validUrl.isUri(url)){
+      return true;
+    } else {
+      return false;
+    }
+}
+
+module.exports = {isValidUrl}
```
Comment (manekenpix) [inline src/utils.js]: Missing newline at the end of the file

File: test/utils.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const utils = require('../src/utils');
+
+
+test('test for URL validation', () => {
+    expect (utils.isValidUrl('https://github.com/Seneca-CDOT/telescope/issues/74')).toBe(true);
+    expect (utils.isValidUrl('github.com/Seneca-CDOT/telescope/issues/74')).toBe(false);
+    expect (utils.isValidUrl('https//github.com/Seneca-CDOT/telescope/issus/74')).toBe(false);
+})
```
Comment (manekenpix) [inline test/utils.test.js]: Missing newline at the end of the file

File: src/utils.js (hunk 1 above)
Comment (Cadesh) [reply src/utils.js]: newline added.

File: test/utils.test.js (hunk 2 above)
Comment (Cadesh) [reply test/utils.test.js]: newline added

**Comment by manekenpix** [comment]
@Cadesh There's still a conflict with `package-lock.json`.

**Comment by humphd** [comment]
```
git checkout master
git pull upstream master
git checkout issue-74
git rebase master
npm install
git add package-lock.json
```

File: test/utils.test.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const utils = require('../src/utils');
+
+test('test for URL validation', () => {
```
Comment (humphd) [inline test/utils.test.js]: Let's break these up into separate tests with a description for each of what is tested.

**Comment by humphd** [comment]
Looks like `package.json` is out of date again, needs another rebase.

**Comment by Cadesh** [comment]
Actually the only thing I did was to manually fix the package.json conflict on GitHub that, by the way, was the same one I fixed and pushed earlier on class. I do not know why the same conflict showed up again.

**Comment by humphd** [comment]
@Cadesh that will do the merge (fixing on GitHub).  You want to do it locally.

I'm going to merge this, since it's not a big deal, and I want this landed.

**Pull request merged**